lib/debug/variable_inspector.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module DEBUGGER__
+  class VariableInspector
+    class Member
+      attr_reader :name, :value
+
+      def initialize(name:, value:, internal: false)
+        @name = name
+        @value = value
+        @is_internal = internal
+      end
+
+      def internal?
+        @is_internal
+      end
+
+      def self.internal name:, value:
+        new(name:, value:, internal: true)
+      end
+    end
+
+    def indexed_members_of obj, start:, count:
+      (start ... (start + count)).map { |i| Member.new(name: i.to_s, value: obj[i]) }
+    end
+
+    def named_members_of obj
+      members = case obj
+      when Array then obj.map.with_index { |o, i| Member.new(name: i.to_s, value: o) }
+      when Hash then obj.map { |k, v| Member.new(name: value_inspect(k), value: v) }
```

Comment:
This used to be inconsistent before. VSCode would format Hash keys with `value_inspect`, but not Chrome. Now they will both us it.